main.tf

```diff
@@ -4,6 +4,7 @@ terraform {
     google = {
       source  = "hashicorp/google"
       version = ">= 3.56.0"
+      configuration_aliases = [ google.alternate ]
```

Comment:
Not sure if `alternate` is some magic suffix. The way I read the docs it needs to explicitly list all aliases? Which would be rather crappy. There's also: https://www.terraform.io/language/modules/develop/providers#passing-providers-explicitly :shrug:

Reply:
oh, then this pr is not needed :)